ubi_manifest/app/api.py

```diff
@@ -10,12 +13,70 @@
 from .models import DepsolveItem, DepsolverResult, DepsolverResultItem, TaskState
 from .utils import get_items_for_depsolving, get_repo_classes
 
+REQUEST_TIMEOUT = 20
+
 router = APIRouter(prefix="/api/v1")
 
 
 @router.get("/status")
-def status() -> dict[str, str]:
-    return {"status": "OK"}
+def status() -> dict[str, Any]:
```

Comment:
Can you please add documentation for this endpoint - at least how expected response looks and maybe response_model? For caller it's difficult to guess what to format of json.
E.g. `Workers status` has a bit complicated structure and in case we use multiple workers, it is currently not obvious for caller how the output looks.